Assets/MixedRealityToolkit/Experimental/Providers/Hands/HandRay/TorsoDirectedHandRay.cs

```diff
@@ -0,0 +1,192 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
```

Comment:
Following examples in [experimental features should go in a separate folder](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/ExperimentalFeatures.html#experimental-code-should-live-in-a-separate-folder), please put this under `MRTK/Experimental/TorsoDirectedHandRay`.

Additionally, since this is an exerimental feature, it should have an [example demonstrating usage](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/ExperimentalFeatures.html#what-to-expect-from-an-experimental-feature) (see bullet point 1).

Reply:
I understand the need for an example.

Would it be possible to fork the MRTK to create a working example?
It is necessary to modify device providers in order to make use of the feature.